ambitious/block-template-parts/content-front-page.html

```diff
@@ -31,3 +31,13 @@ <h1 class="has-white-color has-text-color has-text-align-center intro-heading ha
 	</div>
 </div>
 <!-- /wp:group -->
```

Comment:
I think the content above would be great as a block pattern included in the theme, rather than a template part.

Reply:
I agree, I will add that soon. I have already included the pattern for [GT Ambition](https://wordpress.org/themes/gt-ambition/), the existing PHP version of this theme. Works great.

I used the front-page template to see if it could work as some sort of starter content. I'd really like to have some feature to give users a complete page layout rather than a blank page from the start. Patterns are a huge improvement, but still require users to manually discover and insert them. And they work better for smaller sections, not complete pages. WordPress.com has solved this really well with presenting all sorts of block layouts when a new static page is created.